xml/art_sle_ha_pmremote.xml

```diff
@@ -0,0 +1,426 @@
+  xmlns:xlink="http://www.w3.org/1999/xlink">
+  <title>&paceremote;</title>
+  <subtitle>&productname; &productnumber;</subtitle>
+  <info>
+   <productnumber>&productnumber;</productnumber>
+   <productname>&productname;</productname>
+   <date><?dbtimestamp format="m/d/Y X"?></date>
+   <abstract>
+    <para>
+     This document guides you through the setup of a cluster with virtual
+     guest nodes, managed by &pace; and &pmremote;.
+     <emphasis>Remote</emphasis> in this context does not mean physical
+     proximity, but <quote>non-membership</quote> of a cluster.
+    </para>
+   </abstract>
+  </info>
+
+ <sect1 xml:id="sec.ha.pmremote.usage">
+  <title>Usage Scenario</title>
+  <para>
+   The procedures in this guide describe the process of setting up a minimal
+   cluster with the following characteristics:
+  </para>
+  <itemizedlist>
+   <listitem>
+    <remark>toms 2017-03-23: Is this supported from 12 GA, SP1, or SP2?
+    According to aspiers, since &sls; 12 at least</remark>
+    <para>A cluster node running &productname; &productnumber;. In this guide, its host name is
+     <systemitem class="domainname">&wsI;</systemitem>.</para>
+   </listitem>
+   <listitem>
+    <para>Failover of resources from one node to the other if the active host
+     breaks down (active/passive setup).</para>
+   </listitem>
+   <listitem>
+    <para>Virtual guest nodes running &pmremote;. In this guide, one guest
+     node is named <systemitem class="domainname">&wsII;</systemitem>.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     &pace; to manage guest nodes and remote nodes.
+    </para>
+   </listitem>
+  </itemizedlist>
+ </sect1>
+
+  <sect1 xml:id="sec.ha.pmremote.concept">
+   <title>Conceptual Overview and Terminology</title>
+   <!-- Add a picture? -->
+   <para>
+    A regular cluster may contain up to 32 nodes. With the &pmremote; service,
+    &ha; clusters can be extended to include additional nodes beyond this
+    limit. Remote nodes do not need to have the full cluster stack installed,
+    they only run the &pmremote; service. The service acts as a proxy,
+    allowing the cluster stack on the “regular” cluster nodes to connect to
+    the service. Thus, the node that runs the &pmremote; service is
+    effectively integrated into the cluster as a remote node.
+   </para>
+   <para>Assuming the physical host contains several virtual guest nodes,
+    the standard sequence of &pace; with &pmremote; will be as follows:
+   </para>
+   <orderedlist>
+    <listitem>
+     <para>On the cluster node, virtual machines are launced by &pace;.</para>
+    </listitem>
+    <listitem>
+     <para>The cluster connects to the &pmremote; service of the virtual
+      machines.</para>
+    </listitem>
+    <listitem>
+     <para>The virtual machines are integrated into the cluster by &pmremote;.
+     </para>
+    </listitem>
+   </orderedlist>
+
+   <para>
+    It is important to understand the several distinctive
```

Comment:
'It is important to distinguish between several roles that a virtual machine can take in the &HA; cluster:
- A virtual machine can run the full cluster stack. In this case, the virtual machine is a regular cluster node and is not itself managed by the cluster.
- A virtual machine can be managed by the cluster as a resource, without the cluster being aware of the services that run inside the virtual machine. In this case, the virtual machine is opaque to the cluster. 
- A virtual machine can be a cluster resource, and run pacemaker_remote, which allows the cluster to manage services inside the virtual machine. In this case, the virtual machine is a guest node and is  transparent to the cluster.'